.github/scripts/.build.zsh

```diff
@@ -71,6 +71,7 @@ build() {
   local -i codesign=0
 
   if [[ ${host_os} == linux ]] {
+    local cmake_bin='/usr/bin/cmake'
```

Comment:
`local cmake_bin` needs to be defined outside the conditional (because it will also be used outside of them) and then assigned inside of them.

Reply:
Is it good now ?

Reply:
Yeah that should be fine.